Yosemite/Yosemite/Actions/CardPresentPaymentAction.swift

```diff
@@ -26,6 +26,10 @@ public enum CardPresentPaymentAction: Action {
     ///
     case observeConnectedReaders(onCompletion: ([CardReader]) -> Void)
 
+    /// Get a Stripe Customer for an order.
+    ///
+    case fetchOrderCustomer(siteID: Int64, orderID: Int64, onCompletion: (Result<WCPayCustomer, Error>) -> Void)
```

Comment:
I am not sure about the placement of this method in this store / action. So far, the concern of the store has only been communicating with readers. Creating a customer seems like a different concern. I think we should reconsider this, but for now, let's move forward.

Reply:
> I am not sure about the placement of this method in this store / action. So far, the concern of the store has only been communicating with readers.

Your point is well taken. The thought crossed my mind as well, and I agree that the store has been, until this, reader I/O focused in its actions.